plugins/cache/redis.py

```diff
@@ -100,13 +112,53 @@ def __init__(self, *args, **kwargs):
 
         self._cache = {}
         kw = {}
+
+        # tls connection
         tlsprefix = 'tls://'
         if uri.startswith(tlsprefix):
             kw['ssl'] = True
             uri = uri[len(tlsprefix):]
 
-        connection = uri.split(':')
-        self._db = StrictRedis(*connection, **kw)
+        # redis sentinel connection
+        if self._sentinel_service_name:
+            self._db = self._get_sentinel_connection(uri, kw)
+        # normal connection
+        else:
+            connection = uri.split(':')
+            self._db = StrictRedis(*connection, **kw)
+
+        display.vv('Redis connection: %s' % self._db)
```

Comment:
Can this potentially dump sensitive information (like password)?

Reply:
unlikely, the output, with configured password
```
Redis connection: Redis<ConnectionPool<Connection<host=localhost,port=6379,db=0>>>
```

Reply:
Output of configured redis with password:
```
Redis connection: Redis<SentinelConnectionPool<service=mymaster(master)>
```

What could leak is the Sentinel debug output on line 156
```
Using redis sentinel: [('localhost', '26379'), ('localhost', '26380')] with {'db': '0', 'password': 'changeme'}
```

Reply:
In that case, definitely remove `kw` from that `display.vv` call in line 157.